### PR TITLE
Remove frontmatter

### PR DESCRIPTION
**Pull request by dplanella**
Remove frontmatter altogether to prevent subpages from not building, and other undocumented issues. Vanilla GitHub Pages does not seem to support it.

Remove duplicated how to content. It's already on the subpage (subfolder).

Closes: #19 
Signed-off-by: David Planella <dplanella@linuxfoundation.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
----
```
Comment (jmertic) [inline README.md:1]: I'm wondering if we add an H1 here instead - that would make the markdown readable and should build on GH pages right. Thoughts?

File: README.md [hunk 2]
```diff
@@ -80,43 +75,6 @@ You can reach out to the whole Developer Experience and Community Success team i
 1. [Developer Experience and Community Success Slack channel](https://linuxfoundation.slack.com/messages/devex-and-commsuccess): the team's main communication channel for discussion and sharing knowledge related to our projects and communities. All members of the team are part of the channel, which is also open to anyone else in the organization who either wants to follow the work, ask questions or simply hang out to join. It is a public Slack channel for LF team members only.
 1. [Community hangout Slack channel](https://linuxfoundation.slack.com/messages/community-hangout): the team's social channel. This is a space to share non-work related stories. What did you do last weekend? Show off your pets, show us your last family holiday pics, memes... In summary, a space to get to know each other by offering a window into what we're doing when we're not in front of the computer working. All team members are included by default, but it's voluntary to join/leave the channel. Feel free to share as much or as little as you like.
 
-## How to edit the handbook
```
Comment (jmertic) [inline README.md:83]: Good idea to split this out.

Comment (jmertic) [reply README.md:83]: Actually - it looks like #27 does something similar. Should we revert this one?

Comment (dplanella) [reply README.md:83]: Let's do that. I'm used to using stacked MRs in GitLab, but GitHub does not seem to have this function. I added a comment on the PR's description, but functionally it's not supported.

File: README.md (hunk 1 above)
Comment (dplanella) [reply README.md:1]: I'm not sure I understand, could you elaborate? How would a H1 header help?

**Comment by dplanella** [comment]
Closing as per https://github.com/linuxfoundation/devex-and-commsuccess-handbook/pull/25#discussion_r938003638

**Pull request closed without merging**